index.html

```diff
@@ -0,0 +1,120 @@
+<DOCTYPE! html>
+<html>  
+  <head>
+    <!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="styles.css">    
+  </head>
+  <body>
+    <!-- placeholders for everything held in navbar -->
+    <nav>
+      <div class="container">
+        <div class="row">
+          <div class="col-xs-2 navtitle"><p>Project Name</p></div>

```

Comment:
The secret to awesome collapsable nav bars appears to be: Just copy the whole shebang from the Bootstrap site and delete the parts you don't need!